ec/acc/deployment_basic_test.go

```diff
@@ -63,24 +61,18 @@ func TestAccDeployment_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(resName, "traffic_filter.#", "0"),
 				),
 			},
-			// Ensure that no diff is generated.
-			{Config: cfg, PlanOnly: true},
```

Comment:
Not opposed to this change, but wondering why we are removing these?

Reply:
They're not necessary. I forgot to why I did it in the provider.

I added this at the beginning when writing the provider tests because I thought that TF didn't refresh the state after apply, which it does automatically now by design. So this step is already "happening" after every apply and if there's a plan diff, an error is returned, which has happened to me before.